apps/api/src/test-data/buildUserTestData.ts

```diff
@@ -22,6 +22,34 @@ const dtos: DTO<CoscradUser>[] = [
         username: 'cool-james',
         roles: [CoscradUserRole.projectAdmin],
     },
+    {
+        type: AggregateType.user,
+        id: '2',
+        authProviderUserId: 'auth0|6407b7bd81d69faf23e9dd7e',
+        profile: new CoscradUserProfile({
+            name: {
+                firstName: 'COSCRAD',
+                lastName: 'Test-User',
+            },
+            email: 'cypress-user@coscrad.org',
+        }),
+        username: 'cscrad-cypress-user',
```

Comment:
Note that I fixed this on my branch so we might have a merge conflict at some point due to this. Just wanted to point it out in case you're the one to resolve it.